src/Controls/src/Core/Accelerator.cs

```diff
@@ -67,12 +67,17 @@ public static Accelerator FromString(string text)
 
 			if (text != Separator.ToString())
```

Comment:
We own the `Separator` field, so we might as well make it a string to avoid all the ToString.